resources/backup_scripts/db-backup.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
@kostajh I thought we would modify whatever is currently creating the DB dumps, but it would be neater to have that wrapped up in our db-backup.py script as well. I'll work on that today.